src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -129,13 +137,36 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
   }, [state]);
 
   const onSubmit = useCallback(
-    async (vals: Partial<FormValues>) => {
-      if (!user) return;
+    async (
+      vals: Partial<FormValues>,
+      userStatuses: UserStatus[],
+      showUserStatuses: boolean
+    ) => {
+      if (!user || formError) return;
       try {
         // unfortunately the typing is off for react-hook-forms.
-        if (!!venueId)
+        if (!!venueId) {
           await updateVenue({ ...(vals as VenueInput), id: venueId }, user);
-        else await createVenue(vals as VenueInput, user);
+
+          if (!!sovereignVenueId)
```

Comment:
is there a specific need for the double exclamation mark `!!` ?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0c1f9f07a5696848014d78ad2604de3c1ed2d194